app/views/layouts/application.html.erb

```diff
@@ -15,10 +15,16 @@
             <ul class="nav pull-right">
               <li><%= link_to "Shelter Index", "/shelters" %></li>
               <li><%= link_to "Pet Index", "/pets" %></li>
+              <li>Favorite Pets: <%= favorites.total_count %></li>
             </ul>
           </nav>
       </div>
     </header>
+    <% flash.each do |type, message| %>
+      <section class=<%= type %>>
+        <p><%= message %></p>
+      </section>
+    <% end %>
```

Comment:
I also added this on my branch, so we might get a merge conflict if we added it in slightly different locations. Just a heads up -- will not be a problem to fix at all!